### PR TITLE
Fix typo in test suite

### PR DESCRIPTION
**Pull request by xingyzt**
`.true` -> `.true.`

### CODE REVIEW

File: star/test_suite/20M_pre_ms_to_core_collapse/inlist_common [hunk 1]
```diff
@@ -190,7 +190,7 @@
       limit_for_rel_error_in_energy_conservation = 1d-7
       hard_limit_for_rel_error_in_energy_conservation = 1d-6      
       
-      never_skip_hard_limits = .true
+      never_skip_hard_limits = .true.
```
Comment (pmocz) [inline star/test_suite/20M_pre_ms_to_core_collapse/inlist_common:193]: Great catch!

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -2,31 +2,6 @@
 Changelog
 *********
 
-Changes in main
```
Comment (pmocz) [inline docs/source/changelog.rst:5]: Should we add this back in? In theory it's a good place for developers to write down and accumulate changes for the next release version, although in practice I don't know how often it get used?

Comment (Debraheem) [reply docs/source/changelog.rst:5]: no, we should only include the singular last commit related to the typo fix.

Comment (Debraheem) [reply docs/source/changelog.rst:5]: Earl's commits for release should not be included.

**Comment by Debraheem** [comment]
to get past the issue of you having added old commits from Earl's release branch, I've made a seperate branch and pull request, where I manually added the fix you've found. Thank you! See https://github.com/MESAHub/mesa/pull/679

**Pull request closed without merging**